testdata/testdata.go

```diff
@@ -18,9 +18,14 @@
 package testdata
 
 import (
-	"log"
-	"os"
 	"path/filepath"
+	"runtime"
+)
+
+var (
+	_, b, _, _ = runtime.Caller(0)
```

Comment:
It doesn't really matter in such a small package, but in general: I don't like global variables that are a single letter.  Also it's only an intermediate involved in setting another global.  We could do something like this instead to hide it:

```go
func caller() string { _, b, _, _ = runtime.Caller(0); return b }

var basepath = filepath.Dir(caller())
```

or

```go
var basepath string
func init() { // set basepath }
```

Reply:
Fixed. Doing `init()`